webpack.config.js

```diff
@@ -31,12 +21,60 @@ module.exports = {
         test: /\.(ts|js)x?$/,
         // exclude: /node_modules/,
         loader: "babel-loader",
+        options: { presets: ["@babel/env", "@babel/preset-react"] },
```

Comment:
I don't think we want preset-react for the non-react code.

Reply:
Moved to just react config.